tests/v2/validation/nodescaling/scaling_node_driver_rke1_test.go

```diff
@@ -0,0 +1,70 @@
+package scaling
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioninginput"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type RKE1NodeScalingTestSuite struct {
+	suite.Suite
+	client             *rancher.Client
+	standardUserClient *rancher.Client
+	session            *session.Session
+	clustersConfig     *provisioninginput.Config
+}
+
+func (s *RKE1NodeScalingTestSuite) TearDownSuite() {
+	s.session.Cleanup()
+}
+
+func (s *RKE1NodeScalingTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	s.session = testSession
+
+	s.clustersConfig = new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, s.clustersConfig)
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(s.T(), err)
+
+	s.client = client
+
+	enabled := true
+	var testuser = namegen.AppendRandomString("testuser-")
+	var testpassword = password.GenerateUserPassword("testpass-")
+	user := &management.User{
+		Username: testuser,
+		Password: testpassword,
+		Name:     testuser,
+		Enabled:  &enabled,
+	}
+
+	newUser, err := users.CreateUserWithRole(client, user, "user")
+	require.NoError(s.T(), err)
+
+	newUser.Password = user.Password
+}
+
+func (s *RKE1NodeScalingTestSuite) TestScalingRKE1NodePools() {
+	clusterID, err := clusters.GetClusterIDByName(s.client, s.client.RancherConfig.ClusterName)
+	require.NoError(s.T(), err)
+
+	ScalingRKE1NodePools(s.T(), s.client, clusterID, s.clustersConfig)
```

Comment:
My only gripe here is I don't want scaling to rely on the provisioning input config. If we want dynamic test cases that is fine, but they should be in its own config entry. IMO I think we just have static test cases on what we're scaling. That way we're testing something consistently. In other words static before dynamic.

Reply:
Heard, I will add static test cases in tandem with dynamic tests.